deprecation.py

```diff
@@ -42,7 +44,7 @@ class DeprecatedWarning(DeprecationWarning):
 
     :param function: The function being deprecated.
     :param deprecated_in: The version that ``function`` is deprecated in
-    :param removed_in: The version that ``function`` gets removed in
+    :param removed_in: The version or date that ``function`` gets removed in
```

Comment:
"The version that ``function`` gets removed in, or the `datetime.date` it gets removed on" might be clearer.

Reply:
I'll make the change

Reply:
Fixed